web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -26,8 +26,10 @@ export interface SidebarCardsListProps {
 
 export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement => {
   const [hiddenAccordionIsOpen, setHiddenAccordionIsOpen] = useState<boolean>(false);
+  const [accordionPosition, setAccordionPosition] = useState<number>(0);
```

Comment:
I added the accordion position so we can manage it so when the accordion closes, there isn't that empty space.